persistence-modules/hibernate-types/README.md

```diff
@@ -0,0 +1,7 @@
+## Hibernate Types
```

Comment:
Should I remove this file?

Reply:
Each article in the persistence-module has a README.md file.
This file is also where the instructions for running docker are documented. 
If we remove this file where should the docker directions go?

Reply:
these files should only have a link to the articles inside them, they are managed by the people pushing the buttons to go live etc. as for the instructions, being in the article is sufficient, not worries about that

Reply:
Removed